scripts/lib/aws/elasticache.sh

```diff
@@ -201,6 +225,7 @@ aws_wait_replication_group_available() {
     print_k8s_ack_controller_pod_logs
     exit 1
   fi
+  reload_temp_credentials
```

Comment:
in a future PR, you might consider checking the controller logs for some error message about credentials expiring and then triggering the reload function only when that happens... for now, just brute-forcing reloading here and throughout is fine though.

Reply:
sure.